example/src/App.tsx

```diff
@@ -24,30 +25,44 @@ import {
   Card,
   ClickableChip,
   Loading,
+  Modal,
 } from '../../src/components';
 
 interface State {
   appName?: string,
   appDescription: string,
-
   isMenuOpened: boolean,
+  show: boolean,
   animation?: OffCanvasAnimationType,
-
   appTextCounter: string
 }
 
 class App extends React.Component<{}, State> {
+
   constructor(props: any) {
     super(props);
     this.state = {
       appName: '',
       appDescription: '',
-
       isMenuOpened: false,
+      show: false,
       appTextCounter: ''
     };
   }
 
+  animateIn(modal: object, dialog: object) {
```

Comment:
`modal: object, dialog: object` are not good enough.  They need to be typed through function definition.